.firebaserc

```diff
@@ -1,8 +1,8 @@
 {
   "projects": {
-    "default": "",
+    "default": "sp-open-poll",
```

Comment:
@Roubean1 Please revert this change - it needs to be kept empty so other's can take the repo and configure it as they want